### PR TITLE
Docs: Clarify icon libraries supported on ui-switch

### LINKED ISSUE #531: Icons in switch widgets
I use several ui-switch widgets with icons, but on Dashboard2 icons are not showing...

i.e. - not working:
lightbulb_outline (material icon)
fa-fire (font awesome)
fa-lightbulb-o (font awesome)

### PR DESCRIPTION
**Pull request by joepavitt**
## Description

Closes #531

### CODE REVIEW

File: docs/nodes/widgets/ui-switch.md [hunk 1]
```diff
@@ -5,8 +5,8 @@ props:
     Label: The text shown within the button.
     On Payload: The type & value to output in <code>msg.payload</code> when the switch is turned on.
     Off Payload: The type & value to output in <code>msg.payload</code> when the switch is turned off.
-    On Icon: If provided, this <a href="https://pictogrammers.com/library/mdi/" target="_blank">Material Design icon</a> will replace the default switch when in "on" state
-    Off Icon: If provided, this <a href="https://pictogrammers.com/library/mdi/" target="_blank">Material Design icon</a> will replace the default switch when in "off" state
+    On Icon: If provided, this <a href="https://pictogrammers.com/library/mdi/" target="_blank">Material Design icon</a> will replace the default switch when in "on" state. No need to include the <code>mdi</code> prefix.
+    Off Icon: If provided, this <a href="https://pictogrammers.com/library/mdi/" target="_blank">Material Design icon</a> will replace the default switch when in "off" state. No need to include the <code>mdi</code> prefix.
```
Comment (Steve-Mcl) [inline docs/nodes/widgets/ui-switch.md:9]: I understand the need for clarity but should we not also support mdi (as in `mdi-copy`)?

Just thinking of a future where we permit `fa-copy` and `fas-copy`?

File: nginx.conf [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        proxy_set_header Connection "upgrade";
+        proxy_set_header Host $host;
+
+	    proxy_pass http://host.docker.internal:1880/dashboard/;
+
+        proxy_http_version 1.1;
+        proxy_set_header Upgrade $http_upgrade;
+    }
+  }	
+}
```
Comment (Steve-Mcl) [inline nginx.conf:39]: Did we mean to include an nginx conf in a clarification about icons?

File: docs/nodes/widgets/ui-switch.md (hunk 1 above)
Comment (joepavitt) [reply docs/nodes/widgets/ui-switch.md:9]: We do already support that - but just pointing out it's not required

File: nginx.conf (hunk 2 above)
Comment (joepavitt) [reply nginx.conf:39]: woops

File: docs/nodes/widgets/ui-switch.md (hunk 1 above)
Comment (joepavitt) [reply docs/nodes/widgets/ui-switch.md:9]: The `mdi` library is far larger than `fa` anyway, so I don't think we'll need to venture away anytime soon

File: nodes/widgets/locales/en-US/ui_switch.html [hunk 3]
```diff
@@ -11,10 +11,8 @@
         toggle switch will be replaced with the relevant icons and their respective colors (if defined).
     </p>
     <p>
-        The <b>On/Off Icon</b> field can be either a <a href="https://klarsys.github.io/angular-material-icons/" target="_blank">Material Design icon</a>
-        <i>(e.g. 'check', 'close')</i> or a <a href="https://fontawesome.com/v4.7.0/icons/" target="_blank">Font Awesome icon</a>
-        <i>(e.g. 'fa-fire')</i>, or a <a href="https://github.com/Paul-Reed/weather-icons-lite/blob/master/css_mappings.md">Weather icon</a>.
-        You can use the full set of google material icons if you add 'mi-' to the icon name. e.g. 'mi-videogame_asset'.
+        The <b>On/Off Icon</b> field should be a <a href="https://pictogrammers.com/library/mdi/" target="_blank">Material Design icon</a>
+        <i>(e.g. 'wifi', 'home')</i>.
```
[the remark this answers is not in the document]
Comment (joepavitt) [reply nodes/widgets/locales/en-US/ui_switch.html:15]: The mdi library is far larger than fa anyway, so I don't think we'll need to venture away anytime soon

File: docs/nodes/widgets/ui-switch.md (hunk 1 above)
Comment (Steve-Mcl) [reply docs/nodes/widgets/ui-switch.md:9]: understood. So copy will use `mdi-copy` and (if sometime fown the road we support fas, that would be fas-copy)

File: nodes/widgets/locales/en-US/ui_switch.html (hunk 3 above)
[the remark this answers is not in the document]
Comment (joepavitt) [reply nodes/widgets/locales/en-US/ui_switch.html:15]: in the future, _if_ we include other libraries, then that will come with a docs change to reflect it. Currently, these docs are perfectly accurate and reflect the state of the library

File: docs/nodes/widgets/ui-switch.md (hunk 1 above)
Comment (joepavitt) [reply docs/nodes/widgets/ui-switch.md:9]: yep

**Review by Steve-Mcl — APPROVED** [review]

**Pull request merged**